tests/src/main/java/net/neoforged/neoforge/debug/client/AmbientOcclusionElementsTest.java

```diff
@@ -51,10 +51,10 @@ public AmbientOcclusionElementsTest() {
 
     private void addCreative(BuildCreativeModeTabContentsEvent event) {
         if (event.getTabKey() == CreativeModeTabs.BUILDING_BLOCKS) {
-            event.accept(AO_BLOCK_SHADE_ITEM);
-            event.accept(AO_BLOCK_NO_SHADE_ITEM);
-            event.accept(NO_AO_BLOCK_SHADE_ITEM);
-            event.accept(NO_AO_BLOCK_NO_SHADE_ITEM);
+            event.accept((ItemLike) AO_BLOCK_SHADE_ITEM);
+            event.accept((ItemLike) AO_BLOCK_NO_SHADE_ITEM);
+            event.accept((ItemLike) NO_AO_BLOCK_SHADE_ITEM);
+            event.accept((ItemLike) NO_AO_BLOCK_NO_SHADE_ITEM);
```

Comment:
This looks particularly painful to deal with. Is there no way to get rid of the cast?

Reply:
The casts are redundant indeed.